### ISSUE TITLE
writer.write doesn't work

### ISSUE THREAD

**Issue by Hobyrr** [dragon66/android-gif-animated-writer#2]
Hi ! I know this code is old, but i'm interested so :)

I tried to run your code to make a simply gif with two 20x20png, but the programm fail each time at the same line. (line 48 below) And i've just copy and paste your exemple ... :(  

![image](https://user-images.githubusercontent.com/37924392/57518640-c51e8880-7319-11e9-92ef-a10d5526ff6c.png)

If you have any idea !

**Comment by dragon66** [comment]
@Hobyrr what is the error message? Did you debug it? Without error message, stack trace, it is hard to say what’s going wrong.

Also, make sure bmp1 and bmp2 are decoded correctly and try to write a single gif image instead of an animated gif. This way, it is easier to find the issue. 

By the way, in your screenshot, why can I see some of the method arguments like logicalScreenWidth etc.? Is that a feature of your IDE?

**Comment by Hobyrr** [comment]
@dragon66  Sorry i wasn't available this weekend, but really thx for your fast reply !

After a lot of research, i found the problem, a simple rights access problem ... My bad, not yours at all, wrong gradle configurations.

Your programm run properly and really faster than i thought ! I tried some other codes, but they were slow... So thx to share it ! 

For the method arguments, it's my IDE (visual studio) which automatically print them.

I have a last problem... When i try to read the GIF, it works, but for the first iteration, the second pictures doesn't appear. 
I have 4 pictures (c1,c2,c3,c4), which should print in this order, but for the first iteration, i've c1>c3>c4.
For all next i've c1>c2>c3>c4.

![image](https://user-images.githubusercontent.com/37924392/57615296-a7505e00-757b-11e9-8966-fa5d1c81aed7.png)

If you want to see the GIF, here it is ! If you've any idea... Thx again :) 
![animated_500x500](https://user-images.githubusercontent.com/37924392/57615118-38730500-757b-11e9-9d33-58802bfa9a05.gif)

**Comment by dragon66** [comment]
@Hobyrr Great you figured out the issue and thanks for the following comments:

> Your programm run properly and really faster than i thought ! I tried some other codes, but they were slow... So thx to share it !

For the other issue you mentioned - there is a missing frame in the first iteration, I can't reproduce it. I opened it with both IE and Firefox and they all show it correctly.
 
Sometimes, for the first iteration, playing may not start at frame 0 but it should always be in sequence.

I was able to split the frames of your animated image and I see no issue either.
![split_frame_0](https://user-images.githubusercontent.com/7292053/57624418-df49a800-755f-11e9-9640-54c4b21457c0.gif)
![split_frame_1](https://user-images.githubusercontent.com/7292053/57624356-c0e3ac80-755f-11e9-9a55-40df38ab7fa1.gif)
![split_frame_2](https://user-images.githubusercontent.com/7292053/57624360-c3460680-755f-11e9-878b-2ac9a1156b3d.gif)
![split_frame_3](https://user-images.githubusercontent.com/7292053/57624363-c50fca00-755f-11e9-9583-5ac858c5a886.gif)
